intermediate_source/rpc_tutorial.rst

```diff
@@ -441,11 +441,15 @@ takes a GPU tensor, you need to move it to the proper device explicitly.
         def __init__(self, ntoken, ninp, dropout):
             super(EmbeddingTable, self).__init__()
             self.drop = nn.Dropout(dropout)
-            self.encoder = nn.Embedding(ntoken, ninp).cuda()
+            self.encoder = nn.Embedding(ntoken, ninp)
+            if torch.cuda.is_available():
```

Comment:
we will need to update the text above as well, it currently says "we intentionally put the
``Embedding`` layer on GPU to cover the use case"